build/scripts/build-utils.ps1

```diff
@@ -6,10 +6,16 @@ $ErrorActionPreference="Stop"
 # Import Arcade functions
 . (Join-Path $PSScriptRoot "tools.ps1")
 
-[string]$binariesDir = Join-Path $RepoRoot "Binaries"
+$BinariesConfigDir = Join-Path $ArtifactsDir $configuration # TODO: remove
```

Comment:
Is there a bug tracking this TODO?

Reply:
Working on that (changing the layout) now.

Reply:
https://github.com/dotnet/roslyn/pull/31560